chrome/inject_otp.js

```diff
@@ -0,0 +1,39 @@
+window.browserpassDisplayOTP = function(login) {
+  iframe = document.createElement("iframe");
```

Comment:
One tiny issue: if I run browserpass twice on the same page, it will create two iframes. Then I will never be able to close the one that was opened first.

Could you please assign a unique ID to this iframe (e.g. `browserpass-otp-iframe`), and on the first line here delete an existing element with the same id? `document.getElementById("browserpass-otp-iframe").remove()` should do it I think.

Reply:
Done

Reply:
Yup, tested, thanks! Now only FF issue remains, I hope it's something we could easily workaround, but I can't figure out so far what exactly is missing...

Reply:
I don't use Firefox, so if either of you are able to figure out the issue that would be fantastic.

Reply:
I don't use it too, but I had to install it to test PRs for this extension from time to time 😄 

So far I can only see that `receiveMessage()` in `inject_otp.js` is never being called, and I checked that according to docs the `postMessage` should be supported since forever. There are no errors in the console 😞